res/controllers/MVave-SMK-25-II-scripts.js

```diff
@@ -0,0 +1,277 @@
+                type: components.Button.prototype.types.push,
+            });
+            this.backButton = new components.Button({
+                group: "[Channel1]",
+                midi: [0x90, 0x5B],
+                key: "beatjump_backward",
+            });
+            this.forwardButton = new components.Button({
+                group: "[Channel1]",
+                midi: [0x90, 0x5C],
+                key: "beatjump_forward",
+            });
+            // Faders
+            this.pitchStrip = new components.Pot({
+                group: "[Channel1]",
+                midi: [0xE0, 0xE0],
+                key: "rate",
+            });
+            this.modStrip = new components.Pot({
+                group: "[Channel1]",
+                midi: [0xB0, 0x01],
+                key: "volume",
+            });
+        }
+    }
+
+    class Controller extends components.ComponentContainer {
+        constructor() {
+            super({});
+            this.activeDeck = new Deck();
+            this.gainKnob = new components.Encoder({
+                group: "[Master]",
+                midi: [0xB0, 0x25],
+                key: "gain",
+            });
+            this.headGainKnob = new components.Encoder({
+                group: "[Master]",
+                midi: [0xB0, 0x24],
+                key: "headGain",
+            });
+            this.xfadeKnob = new components.Encoder({
+                group: "[Master]",
+                midi: [0xB0, 0x23],
+                key: "crossfader",
+            });
+            this.recordButton = new components.Button({
+                group: "[Recording]",
+                midi: [0x90, 0x5F],
+                inKey: "toggle_recording",
+                outKey: "status",
+            });
+            // Deck Select
+            this.deckLeftButton = new components.Button({
+                group: "[Channel1]",
+                midi: [0x90, 0x2E], // << Channel Left Button
+                inToggle: function() {
+                    SMK25II.controller.activeDeck.setCurrentDeck("[Channel1]");
+                },
+            });
+            this.deckRightButton = new components.Button({
+                group: "[Channel2]",
+                midi: [0x90, 0x2F], // >> Channel Right button
+                inToggle: function() {
+                    SMK25II.controller.activeDeck.setCurrentDeck("[Channel2]");
+                },
+            });
+            // Drum Pads
+            this.samplers = [];
+            for (let i = 0; i < 16; i++) {
+                this.samplers[i] = new components.SamplerButton({
+                    number: i + 1,
+                    volumeByVelocity: true,
+                });
+            }
+        }
+    }
+
+    SMK25II.init = function() {
+        SMK25II.controller = new Controller();
+    };
+    SMK25II.shutdown = function() {
+        SMK25II.controller.shutdown();
+    };
+
+    const MMCHeader = [0xF0, 0x35, 0x59];
```

Comment:
as far as I can tell, this is just sysex (MMC messages start with `0xF0, 0x7F`), so you may want to change the name

Reply:
Oops, yup, fixed!